FirebaseSessions/Sources/SessionStartEvent.swift

```diff
@@ -42,6 +42,7 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
 //    proto.application_info.development_platform_name;
 //    proto.application_info.development_platform_version;
 
+    proto.application_info.which_platform_info = FIRSESGetAppleApplicationInfoTag()
```

Comment:
This field is a oneof field. What this line does is indicate we're using apple_platform_info instead of android_platform_info so that nanopb can encode correctly. Let me add a comment

Reply:
Moved to a new PR: https://github.com/firebase/firebase-ios-sdk/pull/10459/files